custom-words.txt

```diff
@@ -2236,6 +2236,8 @@ projecthilo
 propagations
 propogation
 protectable
+protobuf
+Protobuf
```

Comment:
Is custom-words really case sensitive?

Reply:
It seems to be. It had both of these pop up in separate errors. I notice that there are other case sensitive words here as well: https://github.com/Azure/azure-rest-api-specs/blob/b1c089bf4cfcdb60add561a4724559d275f13d6d/custom-words.txt#L3618

Reply:
will test

Reply:
I sit corrected, removed and it passes.